src/commands/bindle.rs

```diff
@@ -1,21 +1,11 @@
-use crate::parse_buildinfo;
+use crate::{opts::*, parse_buildinfo};
```

Comment:
Is this supposed to be in this PR?

Reply:
nm I get it; not sure the `*` is worth it here.

Reply:
There are a lot of opts in this file, I'd rather not import every single one..